doc/radosgw/keycloak.rst

```diff
@@ -125,3 +125,104 @@ The tags can be verified using token introspection of the JWT. The command to in
     -d "token=$KC_ACCESS_TOKEN" \
     "http://$KC_SERVER/$KC_CONTEXT/realms/$KC_REALM/protocol/openid-connect/token/introspect" \
     | jq .
+
+Example showing how to fetch a web token from Keycloak using Kerberos or Username and Password authentication
+=============================================================================================================
+
+The following python script takes various command line arguments to authenticate against Keycloak via Kerberos or Basic authentication to obtain a Java Web 
+Token. This token can then be exchanged for temporary credentials with the RADOS gateway in order to AssumeRoleWithWebIdentity:
+
+.. code-block:: python
+
+    #!/usr/bin/env python
+    import json,requests,urllib,re,argparse,sys,os,boto3
+    from requests_kerberos import HTTPKerberosAuth, OPTIONAL
```

Comment:
requests-kerberos is not well-maintained upstream and Fedora is dropping it entirely. requests-gssapi is an up-to-date replacement.